src/appShell/App/Container.tsx

```diff
@@ -0,0 +1,44 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import PageHeader from '../../pages/pageHeader/PageHeader';
+
+import '../../globalStyles/global.scss';
+
+interface IContainerProps {
+    location: Location;
+    children: React.ReactNode;
+}
+
+export default class Container extends React.Component<IContainerProps, void> {
+
+    static contextTypes = {
+        router: React.PropTypes.object,
+    };
+
+    context: {router: any};
+
+    componentDidMount() {
+        const headerNode = document.getElementById("reactHeader");
+        if (headerNode !== null) {
+            ReactDOM.render(<PageHeader router={this.context.router} currentRoutePath={ this.props.location.pathname } />,
+                headerNode);
+        }
+    }
+    renderChildren() {
+        const childProps = {...this.props};
+        const {children} = this.props;
+        return React.Children.map(children,
+            c => React.cloneElement(c as React.ReactElement<any>, childProps));
+    }
+    render() {
+        let children = this.renderChildren();
+
+        // don't create extra div elements if there is only one child
+        if (children.length == 1)
+            return children[0];
```

Comment:
We just need to set this back to what it was.  I don't think there is a reason that nested parent divs should cause a problem for the layout you are trying to achieve.   I'm happy to help address the underlying layout issue.